src/Core/src/Handlers/View/ViewHandler.cs

```diff
@@ -58,7 +58,10 @@ public abstract partial class ViewHandler : ElementHandler, IViewHandler
 #if WINDOWS || MACCATALYST
 				[nameof(IContextFlyoutElement.ContextFlyout)] = MapContextFlyout,
 #endif
-			};
+#if ANDROID
+				["Initialize"] = (h, v) => ((PlatformView?)h.PlatformView)?.Initialize(v)
```

Comment:
What actually triggers Initialize now? 
Also, any way this could use nameof?

Reply:
I actually meant to remove this call from this PR since I just want this to be testing consolidation. My answer here with Peppers covers `What Actually triggers` https://github.com/dotnet/maui/pull/10751#discussion_r998412727

`SetVirtualView` iterates over the mapper and calls every single function for every single key in the mapper dictionary.

> Also, any way this could use nameof?

Hmm maybe? The Initialize call is sort of just a performance trick we're using that only applies to Android. So, not sure off hand what thing we'd tie it to. It'd be neat if 'setvirtualview' and `connecthandler` just ran through funcs/mappers somehow and we just used that.  I'm going to remove this code from this PR and I'll think on it a little bit more once I do https://github.com/dotnet/maui/pull/10751